packages/react-native-worklets/src/shareables.ts

```diff
@@ -355,6 +372,26 @@ function clonePlainJSObject<T extends object>(
   return clone;
 }
 
+function cloneTurboModule<T extends object>(
```

Comment:
Please also add a comment (TSDoc) what kind of structure `TurboModuleLike` is
```suggestion
function cloneTurboModuleLike<T extends object>(
```